src/components/models/characters/Mario_kart.tsx

```diff
@@ -21,36 +27,31 @@ export function Mario({ currentSpeed, steeringAngleWheels, isBoosting, shouldLau
   const [scale, setScale] = React.useState(1)
   const { actions } = useStore()
   const [shouldSlow, setShouldSlow] = React.useState(false)
```

Comment:
```suggestion
  const [,, setShouldSlow] = React.useState(false)
```

Reply:
`Tuple type '[boolean, Dispatch<SetStateAction<boolean>>]' of length '2' has no element at index '2'.ts(2493)`

Shouldn't that state be removed?